doxia-core/src/main/java/org/apache/maven/doxia/parser/XhtmlBaseParser.java

```diff
@@ -77,9 +77,6 @@
     /** Used to wrap the definedTerm with its definition, even when one is omitted */
     boolean hasDefinitionListItem = false;
 
-    /** Decoration properties, eg for texts. */
-    private final SinkEventAttributeSet decoration = new SinkEventAttributeSet();
```

Comment:
Can you please explain why this is gone and most has been turned into `sink#inline()`?